manageiq-ssh-util.gemspec

```diff
@@ -15,8 +15,12 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,lib}/**/*", "LICENSE.txt", "Rakefile", "README.md"]
 
+  s.add_dependency "activesupport"
   s.add_dependency "net-ssh", "~> 4.2"
+  s.add_dependency "net-sftp"
```

Comment:
@djberg96 Do you know what version this should be or should I leave it open ended?

Reply:
@Fryguy Pretty sure I tested with 2.1.x if you want to pin it to that.